runner/ansible/roles/2.1.8/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.8'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # make sure sudoers is set up
+    grep -E "[a-z][a-z0-9]{2}adm +ALL=\(ALL\) +NOPASSWD: +/usr/sbin/crm_attribute +-n +hana_[a-z][a-z0-9]{2}_site_srHook_\*" /etc/sudoers && exit 0
```

Comment:
A lot of duplicated texts. Let's try to move them to variables

Reply:
Give it a shot. I would need to be on my home machine to do that.